my-app/META-INF/RASCAL.MF

```diff
@@ -1,5 +1,6 @@
 Manifest-Version: 0.0.1
 Main-Function: main
 Project-Name: my-app
+Require-Libraries: |lib://rascal-lsp|, |lib://frink|, |lib://frinklang|, |lib://local-maven-repo|
```

Comment:
These don't seem to make a difference; `rascal-lsp` is detected by default, and the other 3 dependencies cannot be resolved `because it does not seem to exist as a project or a library`